### PR TITLE
Tyler django lab 1

### PR DESCRIPTION
**Pull request by CardinalKibbles**

### CODE REVIEW

**Review by cdmichaelb — CHANGES_REQUESTED** [review]
Please add some more formatting to the page using bootstrap. 


Here are a couple of bootstrap classes you could pull from:

[form-control](https://getbootstrap.com/docs/5.0/forms/form-control/)

[list-group](https://getbootstrap.com/docs/5.0/components/list-group/)

**Comment by CardinalKibbles** [comment]
I updated the visuals and made it more usable

**Review by cdmichaelb — APPROVED** [review]
Looks much better! Good job.

File: code/tyler/django/django_lab-1/todo_app/models.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
+from tkinter import CASCADE
+from django.db import models
+from django.forms import CharField
```
Comment (perennialAutodidact) [inline code/tyler/django/django_lab-1/todo_app/models.py:3]: Be careful with using `tab` to autocomplete. Recent updates to VSCode will perform automatic imports on things. Lines 1 and 3 are irrelevant in this case

**Review by perennialAutodidact — APPROVED** [review]

**Pull request merged**